netskope legacy collection (#1426)
SumoLogic/sumologic-documentation: docs/integrations/security-threat-detection/netskope.md

## Collect Logs for Netskope

> To collect logs from the Netskope platform, use the [new Cloud to Cloud Integration for Netskope](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/Netskope-Source) to create the source and use the same source category while installing the app. For the legacy collection, see [Old Collection method](/docs/integrations/security-threat-detection/netskope-legacy-collection/).

Reviewer: ```suggestion
To collect logs from the Netskope platform, use the [new Cloud to Cloud Integration for Netskope](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/Netskope-Source) to create the source and use the same source category while installing the app. You can view the Netskope legacy collection method [here](/docs/integrations/security-threat-detection/netskope-legacy-collection/).
```
